spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -160,12 +160,24 @@ def start_check_update(self, startup=False):
         """
         Check for Spyder updates using a QThread.
 
+        Do not check for updates if the environment is a system or a managed
+        environment.
+
         Update actions are disabled in the menubar and statusbar while
         checking for updates.
 
         If startup is True, then checking for updates is delayed 1 min;
         actions are disabled during this time as well.
         """
+        if (
+            sys.executable.startswith(('/usr/bin/', '/usr/local/bin/'))
+            or False  # TODO: replace with test for distro file
```

Comment:
According to this [doc page](https://packaging.python.org/en/latest/specifications/externally-managed-environments/#marking-an-interpreter-as-using-an-external-package-manager), you need to look for a file named `EXTERNALLY-MANAGED` in `sysconfig.get_path("stdlib", sysconfig.get_default_scheme())`.

But we need to exclude conda envs from this check, otherwise we won't be able to update Spyder in them.